.web5-spec/main.ts

```diff
@@ -0,0 +1,51 @@
+import express from "express";
+import { credentialIssue } from "./credentials.js";
+import { didIonCreate } from "./did-ion.js";
+import { encoderBase58Decode, encoderBase58Encode, encoderBase64Decode, encoderBase64Encode, encoderSha256Encode } from "./encoders.js"
+import type * as http from "http";
+import type { Request, Response } from "express";
+import { paths } from "./openapi.js"; // generated with npx openapi-typescript .web5-component/openapi.yaml -o .web5-component/openapi.d.ts
+import bodyparser from "body-parser";
+
+const app: express.Application = express();
+app.use(express.json());
+app.use(bodyparser.json());
+
+app.post("/did-ion/create", didIonCreate);
+
+app.post("/credentials/issue", credentialIssue);
+
+app.post("/encoders/base64/encode", encoderBase64Encode);
+app.post("/encoders/base64/decode", encoderBase64Decode);
+// app.post("/encoders/base58/encode", encoderBase58Encode);
+// app.post("/encoders/base58/decode", encoderBase58Decode);
```

Comment:
Looks like there are methods in `encoders.ts` for these, should they be uncommented?

Reply:
ah yes, thanks for catching that! We don't have tests for base58 yet, but we can expose the endpoints for this repo for now.

Reply:
correction: [they don't actually work](https://github.com/TBD54566975/web5-js/actions/runs/6857138972/job/18645661619?pr=291) which is why they were commented out. I've removed them entirely now.